packages/client/modules/userDashboard/components/OrgBilling/BillingForm.tsx

```diff
@@ -20,56 +26,74 @@ const StyledForm = styled('form')({
   alignItems: 'space-between'
 })
 
-const PaymentWrapper = styled('div')({
-  height: 160
-})
-
 const UpgradeButton = styled(PrimaryButton)<{isDisabled: boolean}>(({isDisabled}) => ({
   background: isDisabled ? PALETTE.SLATE_200 : PALETTE.SKY_500,
   color: isDisabled ? PALETTE.SLATE_600 : PALETTE.WHITE,
   boxShadow: 'none',
   marginTop: 16,
   width: '100%',
   elevation: 0,
-  '&:hover': {
+  '&:hover, &:focus': {
```

Comment:
Just a side note, the "Plans" panel does not have any focus behaviour which makes it impossible to use with just the keyboard

Reply:
Yeah, that would be nice! I've created an issue to work on that: https://github.com/ParabolInc/parabol/issues/8085